src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/chatpageobject/ChatPage.java

```diff
@@ -67,38 +89,44 @@
   private static final String PRIVATE_MESSAGE_USER_SELECTOR = "#priv-user-%s";
   private static final String PRIVATE_MESSAGE_SELECTED_USER_SELECTOR =
       "#priv-user-%s.User.selected";
-  private static final String MESSAGE_ON_CHAT = "//span[@class='message'][contains(text(), '%s')]";
+  private static final String PATH_MESSAGE_ON_CHAT = "//span[@class='message'][contains(text(), '%s')]";
   private static final String NOTIFICATION_COUNTER =
       "//span[@class='splotch' and contains(text(), '%s')]";
+  private final String USER_NAME_ON_USER_PROFILE_SCREEN = userNameTitle.getText();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this field "USER_NAME_ON_USER_PROFILE_SCREEN" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00116)